Core/Core/API/APIRequestable.swift

```diff
@@ -59,7 +59,7 @@ public class APIJSONEncoder: JSONEncoder {
     }
 }
 
-public typealias APIFormData = [String: APIFormDatum]
+public typealias APIFormData = [(key: String, value: APIFormDatum)]
```

Comment:
The order of these pairs matters for s3 ("file" must be last) so this needs to be an array and not a dictionary since swift dictionaries have undefined order.